gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_18_0/schema.yml

```diff
@@ -151,8 +151,6 @@ databaseChangeLog:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.